### ISSUE TITLE
Blinding on EQTransformer

### ISSUE THREAD

**Issue by zoekrauss** [seisbench/seisbench#62]
In the EQTransformer model,https://github.com/seisbench/seisbench/blob/c8c260854143e8241219ec25d9436188a7af124c/seisbench/models/eqtransformer.py#L215 a blinding step is performed on the annotated prediction, to cut a specified number of samples from the beginning and end of the trace.

The code describes that this is done because of bias in the training data, e.g. "all picks in the training fall between seconds 5 and 55". However, I do not see anywhere in the EQTransformer documentation that describes this limitation to its training- **why are 500 samples cut from the beginning and end of the traces for EQTransformer specifically?**

The applied blinding can be easily changed by the user by simply modifying the downloaded pretrained model:
model = sbm.EQTransformer.from_pretrained("original")
model.default_args["blinding"] = (0,1)

However, because of how the blinding variable is applied in the annotate_window_post() function:
```python
 def annotate_window_post(self, pred, piggyback=None, argdict=None):
        # Combine predictions in one array
        prenan, postnan = argdict.get("blinding", (0, 0))
        pred = np.stack(pred, axis=-1)
        pred[:prenan] = np.nan
        pred[-postnan:] = np.nan
        return pred
```
It is impossible to define zero blinding, (0,0), such that 0 samples are taken off the beginning and end of the trace, since if postnan=0, then pred[-postnan:] = np.nan will return nans for the entire trace. **Could this be rewritten with an if statement that does not perform the blinding if blinding is defined as (0,0) in argdict?**

**Comment by yetinam** [comment]
Hi Zoe,

thanks for spotting this issue! Indeed, setting `blinding = (0, 0)` should disable the blinding. I'll push a fix in the next minutes.

Now to the question, why we decided to add the blinding in the first place. EQTransformer is originally trained on STEAD, which contains 60 s input traces, the same input length as EQTransformer. Within these traces, phase arrivals never occur before second 5 or after second 55. As EQTransformer has a global view of the trace, it can learn these patterns. To address this issue, during EQTransformer training data augmentation is applied: traces are "rotated" in time. This makes sure phase arrivals can appear at any position in the trace. However, it also introduces a discontinuity from the original difference between first and last sample. We think that EQTransformer is, through this discontinuity, able to pick up, whether a trace has been rotated. Consequently, it likely still has the bias against picks at the border seconds. Therefore, we decided to blind these seconds and rather overlap several windows. We are aware that the original EQTransformer implementation **does not** perform this blinding. It is also not necessary for EQTransformer trained on datasets with longer traces (e.g., INSTANCE, ETHZ, GEOFON), but we still kept it, as pick detection on the trace borders is naturally more difficult.

I hope I was able to answer your question. Feel free to ask for clarifications.
